Model/RatepayInvoicePaymentMethod.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Model;
+
+use Wirecard\Oxid\Core\Helper;
+
+use Wirecard\PaymentSdk\Transaction\RatepayInvoiceTransaction;
+
+/**
+ * Payment method implementation for Ratepay Invoice
+ *
+ * @since 1.2.0
+ */
+class RatepayInvoicePaymentMethod extends PaymentMethod
+{
+    /**
+     * @inheritdoc
+     *
+     * @since 1.2.0
+     */
+    protected static $_sName = "ratepay-invoice";
```

Comment:
So gar we always skipped delimiters in payment names, e.g. "creditcard" or "sepacredit". So maybe we should just call this "ratepayinvoice"?

Reply:
no, this is the name we need for the transaction service.

Reply:
You could use the consts from the transaction. You don't need to change it. Just to maybe think of it if its better.

Reply:
Thanks for the hint, we will consider that 👍